src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiInPort.ts

```diff
@@ -0,0 +1,33 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiInPort {
+		private static dispatchMessage: (serializedMessage: string) => number;
+
+		private static instanceMap: Array;
+
+		public static startMessageListener(encodedDeviceId: string) {
+			if (window.DeviceMotionEvent) {
+				this.dispatchMessage = (<any>Module).mono_bind_static_method("[Uno] Windows.Devices.Midi.MidiInPort:DispatchMessage");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)